modules/orca/variables.tf

```diff
@@ -56,6 +50,13 @@ variable "workflow_config" {
 
 
 ## OPTIONAL - Default variable value is set in ../variables.tf to keep default values centralized.
+variable "aws_profile" {
+  type        = string
+  description = "AWS profile used to deploy the terraform application."
+  default = null
```

Comment:
We would want to set this default in the base variables.tf file in the base of the repo.

Reply:
Didn't even know that was a thing... Don't know why that isn't in the folder with the rest of the TF. Fixed.

Reply:
Has to do with the way the build occurs and making sure all the pathing is correct to the various resources.